yaptide/persistence/models.py

```diff
@@ -95,7 +95,8 @@ class SimulationModel(db.Model):
     update_key_hash: Column[str] = db.Column(db.String,
                                              doc="Update key shared by tasks granting access to update themselves")
     tasks = relationship("TaskModel")
-    estimators = relationship("EstimatorModel")
+    estimators = relationship("EstimatorModel", cascade="delete")
+    inputs = relationship("InputModel", cascade="delete")
```

Comment:
I am curious why there is no `cascade="delete"`  for tasks as well ?
Looking at the datamodel, the tasks should be removed as well. Take a look here:
https://yaptide.github.io/yaptide/persistency/

Reply:
Same for logfiles